test/Npgsql.Tests/AutoPrepareTests.cs

```diff
@@ -21,6 +19,7 @@ public void Basic()
         };
 
         using var conn = OpenConnection(csb);
+        conn.UnprepareAll();
```

Comment:
Is it maybe better to just use a temporary pool per test everywhere here? Another option is to just turn off pooling, though that would make the tests incompatible with multiplexing (which currently isn't been executed anyway)

Reply:
> Is it maybe better to just use a temporary pool per test everywhere here?

We can, but that would lead to constant physical connection open/close, so pretty much the same thing.

Reply:
Sure - it's more about simplifying the test code, in case we forget to do Unprepare etc. Not very important.